### DISCUSSION TITLE
Conditional Naming Prefix?

### DISCUSSION THREAD

**Discussion by CountZeroOr** [comictagger/comictagger#685] [Q&A] [answered]
For renaming, how would I do a conditional naming prefix?

For example, for a volume of a magazine (PC Games Vol. 2 #5), what I'd like to do is have it renamed to "PC Games V2 #5" - but for magazines that don't have a volume number (PC Gamer #118), I'd like to not have the "V" present.

**Comment by lordwelch** [comment, marked as answer]
Enable the text cleanup option
